app/controllers/api/v2/override_values_controller.rb

```diff
@@ -43,6 +43,7 @@ def show
       param_group :override_value, :as => :create
 
       def create
+        params[:override_value].merge!(type: "PuppetLookupValue") if @smart.puppet?
         @override_value = @smart.lookup_values.create!(params[:override_value])
```

Comment:
Will this actually instantiate the right class, or just set the `type` column?  Would Foreman::STI be required?